pinot/metrics.py

```diff
@@ -0,0 +1,37 @@
+""" Metrics to evaluate and train models.
+
+"""
+# =============================================================================
+# IMPORTS
+# =============================================================================
+import dgl
+import torch
+
+# =============================================================================
+# MODULE FUNCTIONS
+# =============================================================================
+def _mse(y, y_hat):
+    return torch.nn.functional.mse_loss(y, y_hat)
+
+def mse(net, g, y):
+    y_hat = net.expectation(g)
+    return _mse(y, y_hat)
+
+def _rmse(y, y_hat):
+    return torch.sqrt(
+            torch.nn.functional.mse_loss(y, y_hat))
+
+def rmse(net, g, y):
+    y_hat = net.expectation(g)
+    return _rmse(y, y_hat)
+
+def _r2(y, y_hat):
+    print(y)
```

Comment:
Noting this one prints its input every time it's called, but the other metrics do not